File: python/prism_sdk/multimodal_lease_fencing_integrity_inference.py
"""Factory P32 multimodal lease/fencing integrity feature."""
from .lease_fencing_integrity_support import *
FEATURE_ID="AFA-factory-P32-F02"; CONTRACT_VERSION="factory-multimodal_lease_fencing_integrity_inference/1.0"
def multimodal_lease_fencing_integrity_inference_manifest():return manifest(feature_id=FEATURE_ID,contract_version=CONTRACT_VERSION,scale="multimodal",mode="inference")
def qualify_multimodal_lease_fencing_integrity_inference(request):return qualify(request,feature_id=FEATURE_ID,contract_version=CONTRACT_VERSION,scale="multimodal",mode="inference")

File: crates/factory/src/multimodal_lease_fencing_integrity_inference.rs
//! Factory P32 multimodal lease/fencing integrity feature.
use super::lease_fencing_integrity_support::{
    manifest, qualify, LeaseFencingIntegrityCard7, LeaseFencingIntegrityError,
    LeaseFencingIntegrityRequest4,
};
pub const FEATURE_ID: &str = "AFA-factory-P32-F02";
pub const CONTRACT_VERSION: &str = "factory-multimodal_lease_fencing_integrity_inference/1.0";
pub fn multimodal_lease_fencing_integrity_inference_manifest() -> serde_json::Value {
    manifest(FEATURE_ID, CONTRACT_VERSION, "multimodal", "inference")
}
pub fn qualify_multimodal_lease_fencing_integrity_inference(
    q: &LeaseFencingIntegrityRequest4,
) -> Result<LeaseFencingIntegrityCard7, LeaseFencingIntegrityError> {
    qualify(q, FEATURE_ID, CONTRACT_VERSION, "multimodal", "inference")
}
